hw/dv/data/tests/stress_tests.hjson

```diff
@@ -3,14 +3,16 @@
     {
       name: "{name}_stress_all"
       uvm_test_seq: "{name}_stress_all_vseq"
-      run_opts: ["+test_timeout_ns=10_000_000_000"]
+      // 10ms
+      run_opts: ["+test_timeout_ns=10000000000"]
```

Comment:
if using `_`, xcelium won't consider it as number

Reply:
Fix this as well?
```console
ip/rv_timer/dv/rv_timer_sim_cfg.hjson:57:      run_opts: ["+zero_delays=1", "+test_timeout_ns=10_000_000_000"]
```

Reply:
Good catch. Updated, thanks.